repos/kaleido/js/src/mermaid/render.js

```diff
@@ -0,0 +1,66 @@
+/* global Mermaid:false */
+
+const semver = require('semver')
+const hasOwn = require('object.hasown')
+
+if (!Object.hasOwn) {
+	hasOwn.shim();
+}
+
+/**
+ * @param {object} info : info object
+ *  - data
+ *  - format
+ *  - width
+ *  - height
+ *  - scale
+ */
+function render (info) {
+
+  // Rename info.data to info.markdown
+  info.markdown = info.data
+  delete info.data;
+  
+  let errorCode = 0
+  let result = null
+  let errorMsg = null
+  let pdfBgColor = null
```

Comment:
nit: this can be const

Reply:
All of these ? But these variables might get changed later in the function 🤔

Reply:
Only pdfBgColor, it doesn't change here